templates/admin/user/show.html.twig

```diff
@@ -38,11 +47,12 @@
         </a>
     </p>
     <br>
-    <p align="center">
-        <a href="{{ path('user_edit', {'id': user.id}) }}">
-            <button class="btn btn-success">Edit</button>
-        </a>
-    </p>
-
-    {{ include('admin/user/_delete_form.html.twig') }}
+    {% if user.username != 'sasa' %}
```

Comment:
Same here - I suppose you need a role check

Reply:
I made it to prevent deleting of my admin user

Reply:
You can use such thing as privelege and as possibility to have account that can't be deleted. Your can create a ticket for that later